openedx/core/djangoapps/theming/tests/test_theme_style_overrides.py

```diff
@@ -35,6 +38,36 @@ def test_footer(self):
         # This string comes from header.html of test-theme
         self.assertContains(resp, "This is a footer for test-theme.")
 
+    @with_comprehensive_theme("edx.org")
+    def test_header_no_mktg_links(self):
+        """
+        Test that theme header doesn't show marketing site links for Account Settings page.
+        """
+        # Login
+        user = UserFactory.create()
+        result = self.client.login(username=user.username, password='test')
+        self.assertTrue(result)
+
+        account_settings_url = reverse('account_settings')
+        resp = self.client.get(account_settings_url)
+        self.assertEqual(resp.status_code, 200)
```

Comment:
This is not necessary. Django's assertNotContains performs the status code check for you: https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.SimpleTestCase.assertNotContains.